Added green btns on cards, popup with select trainings day

KatyEkat opened the pull request.

KatyEkat committed:
Added green btns on cards, popup with select trainings day

Decronus commented on src/components/Card/Card.jsx:
```diff
@@ -1,14 +1,12 @@
 import * as S from './styles'
 
-function Card({ titleCard, cardImage }) {
+function Card({ titleCard, cardImage, isForwardBtn = false, onSelectWorkout }) {
```
Допустимо ли указывать для пропсов значение по умолчанию, и если да, уместно ли это в данном случае? Если мы уберем "= false", будет работать корректно? Ведь если мы ничего не передаем в пропс, то он и так примет значение false, а нас это устраивает.

KatyEkat committed:
removed "false" in props Card comp

KatyEkat committed:
MyProfile page fixing

# Conflicts:
#	src/components/Card/Card.jsx
#	src/components/Card/styles.js
#	src/pages/MyProfile/Myprofile.jsx
#	src/pages/MyProfile/styles.js

KatyEkat commented:
fixing MyProfile page
